src/appleseed/renderer/kernel/rendering/generic/genericsamplerenderer.cpp

```diff
@@ -50,6 +50,7 @@
 #include "renderer/modeling/camera/camera.h"
 #include "renderer/modeling/frame/frame.h"
 #include "renderer/modeling/scene/scene.h"
+#include "renderer/modeling/shadergroup/shadergroup.h"
```

Comment:
This dependency doesn't seem required by the code changes introduced in this PR. Is it required by the unchanged code, i.e. was it missing?